addon/emailTemplates.jsx

```diff
@@ -0,0 +1,63 @@
+this.emailTemplates = (function () {
```

Comment:
I'm not sure I understand? I assume you mean the module pattern we're using here – it's basically what we use in Screenshots, creates a private namespace with explicit exports. `this.emailTemplates` is a slightly weird pattern for creating a global in a WebExtension, preferred over some other patterns because of the nested namespaces in a content script and some of the dangers of re-declaring (which aren't a problem in the background script, but are problems for content scripts).